spec.bs

```diff
@@ -647,6 +647,31 @@ Each time a <{fencedframe}> navigates to a [=fenced frame config=], it is instan
 [=fenced frame config instance=], which governs the particular [=browsing context group=] inside the
 [=fenced navigable container/fenced navigable=].
 
+<h4 id=fenced-frame-config-use-cases>Use cases</h4>
+
+Rendering an ad created through an ad auction:
+
+An ad auction API runs an auction and determines a winning ad. Details about the winning ad must be
+hidden from the embedder, and the embedding context cannot be allowed to influence the environment
+of the <{fencedframe}>. Either of those will allow for information to flow across the fenced frame
+boundary, which can allow for colluding parties to join cross-site data and build a profile on the
+user. To prevent that, the API constructs a [=fenced frame config=] whose underlying [=URL=] is
```

Comment:
What should `[=URL=]` actually be here? Should we point to an internal member in the FencedFrameConfig? Or the URN map?